projects/ng7-mat-breadcrumb/src/lib/breadcrumb.model.ts

```diff
@@ -1,5 +1,5 @@
-
 export interface Breadcrumb {
-    label: string;
-    url: string;
-  }
+  label: string;
```

Comment:
Is now used just for storing what defines the label to be analyzed later to create the displayText